docs/doxygen-user/multi-user/upgradeSolr8.dox

```diff
@@ -1,26 +1,36 @@
-/*! \page upgrade_solr_page Upgrading Solr
+/*! \page upgrade_solr8_page Upgrading to Autopsy 4.18.0 (with Solr 8)
 
 [TOC]
 
-\section solr_upgrade_solr4 Upgrading from Solr 4 from Solr 8
+Autopsy 4.18.0 and beyond will make new cases with Solr 8 instead of Solr 4.  Because Solr 8 is not backward compatable with Solr 4, this will have some impact on you. Notably:
 
-To upgrade from Solr 4 to Solr 8, first follow the instructions on the \ref install_solr_page page. At the end of that process, all new cases will be created using Solr 8. If you need to open older cases that were created using Solr 4, please see the following sections which describe how to set up Solr 4 and Solr 8 in parallel and how to migrate Zookeeper data.
+- Cases made with 4.17.0 and earlier will continue to use Solr 4 indexes and can be opened with Autopsy 4.18.0+
+- Cases made with 4.18.0 and later will use Solr 8 and cannot be opened by Autopsy 4.17.0 and earlier.
+The main take away is that you can open older cases, but new cases can't be opened by older software.
```

Comment:
I thought it was true in general that we do not support opening cases with older versions of Autopsy than they were created with. For example, several of our upgrades rename tables which destroys backwards compatibility. These statements would imply to me that only going from 4.18 to an earlier version would be an issue.